src/components/pages/dashboard/panels/map/mapPanel.js

```diff
@@ -0,0 +1,118 @@
+// Copyright (c) Microsoft. All rights reserved.
+
+import React, { Component } from 'react';
+
+import { Indicator } from 'components/shared';
+import {
+  Panel,
+  PanelHeader,
+  PanelHeaderLabel,
+  PanelContent,
+  PanelOverlay,
+  PanelError
+} from 'components/pages/dashboard/panel';
+
+import './mapPanel.css';
+
+const atlas = window.atlas;
+const deviceLayer = 'devices-layer';
+
+export class MapPanel extends Component {
+
+  componentDidMount() {
+    if (this.props.azureMapsKey) {
+      const seattlePosition = new atlas.data.Position(-122.3320708, 47.6062);
+      this.map = new atlas.Map('map', {
+        'subscription-key': this.props.azureMapsKey,
+        center: seattlePosition,
```

Comment:
based on actual lat/long of devices?

Reply:
This is the default location of map before devices are loaded. If there are not devices it just center the map on Seattle.